dockerfiles/msf4j/Dockerfile

```diff
@@ -16,14 +16,89 @@
 
-# set to product base image
-FROM wso2ei-base:6.2.0
+# set to latest Ubuntu LTS
+FROM ubuntu:16.04
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# set jdk configurations
+ARG JDK_NAME=jdk-8u251-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+ARG JDK_URL
+ARG WSO2_SERVER=wso2ei
+ARG WSO2_SERVER_VERSION=6.2.0
+ARG WSO2_SERVER_REPOSITORY=product-ei
+ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+ARG WSO2_SERVER_DIST_URL
+# build arguments for external artifacts
+ARG MYSQL_CONNECTOR_VERSION=5.1.49
+# build argument for MOTD
+ARG MOTD="\n\
+Welcome to WSO2 Docker resources.\n\
+------------------------------------ \n\
+This Docker container comprises of a WSO2 product, running with its latest GA release \n\
+which is under the Apache License, Version 2.0. \n\
+Read more about Apache License, Version 2.0 here @ http://www.apache.org/licenses/LICENSE-2.0.\n"
+
+# install required packages
+RUN \
+    apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends --no-install-suggests \
+        curl \
+        wget \
+        unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+# installe Oracle JDK
+RUN \
+    mkdir -p ${JAVA_HOME} \
+    && wget --no-check-certificate -O ${JDK_NAME} ${JDK_URL} \
+    && tar -xf ${JDK_NAME} -C ${JAVA_HOME} --strip-components=1 \
+    && chown wso2carbon:wso2 -R ${JAVA_HOME} \
+    && rm -f ${JDK_NAME}
+
+# add the WSO2 product distribution to user's home directory
+RUN \
+    wget --no-check-certificate -O ${WSO2_SERVER}.zip "${WSO2_SERVER_DIST_URL}" \
+    && unzip -d ${USER_HOME} ${WSO2_SERVER}.zip \
+    && chown wso2carbon:wso2 -R ${WSO2_SERVER_HOME} \
+    && mkdir -p ${WORKING_DIRECTORY}/wso2-tmp \
+    && cp -r ${WSO2_SERVER_HOME}/wso2/analytics/conf/analytics ${WORKING_DIRECTORY}/wso2-tmp
+
+# create the non-root user and group and set MOTD login message
+RUN \
+    groupadd --system -g ${USER_GROUP_ID} ${USER_GROUP} \
+    && useradd --system --create-home --home-dir ${USER_HOME} --no-log-init -g ${USER_GROUP_ID} -u ${USER_ID} ${USER} \
+    && echo '[ ! -z "${TERM}" -a -r /etc/motd ] && cat /etc/motd' >> /etc/bash.bashrc; echo "${MOTD}" > /etc/motd
 
 # copy entrypoint bash script to user home
-COPY --chown=wso2carbon:wso2 init.sh ${WORKING_DIRECTORY}/
+COPY --chown=wso2carbon:wso2 docker-entrypoint.sh ${USER_HOME}/
+
+# add MySQL JDBC connector to server home as a third party library
+ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/mysql/mysql-connector-java/${MYSQL_CONNECTOR_VERSION}/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.jar ${WSO2_SERVER_HOME}/lib
+
+# add libraries for Kubernetes membership scheme based clustering
+ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/dnsjava/dnsjava/2.1.8/dnsjava-2.1.8.jar ${WSO2_SERVER_HOME}/lib/
```

Comment:
@RidmiR you don't need to add the Kubernetes Membership Scheme and DNS Java artifacts to MSF4j.